DeviceAdapters/BaslerFrameGrabbers/mE5AQ8-CXP6D/Basler_mE5AQ8-CXP6D_CameraDevice.h

```diff
@@ -0,0 +1,242 @@
+///////////////////////////////////////////////////////////////////////////////
+// FILE:          Basler_mE5AQ8-CXP6D_CameraDevice.h
+// PROJECT:       Micro-Manager
+// SUBSYSTEM:     DeviceAdapters
+//-----------------------------------------------------------------------------
+// DESCRIPTION:   Device Adapter for Basler microEnable5 AQ8-CXP6D
+//                framegrabber board
+//                (formerly: Silicon Software mE5AQ8-CXP6D)
+// COPYRIGHT:
+//                Copyright 2021 BST
+//
+// VERSION:		  1.0.0.1 
+//
+// LICENSE:
+// Redistribution and use in source and binary forms, with or without modification,
+// are permitted provided that the following conditions are met:
+//
+// 1. Redistributions of source code must retain the above copyright notice, this
+// list of conditions and the following disclaimer.
+//
+// 2. Redistributions in binary form must reproduce the above copyright notice, this
+// list of conditions and the following disclaimer in the documentation and/or other
+// materials provided with the distribution.
+//
+// 3. Neither the name of the copyright holder nor the names of its contributors may
+// be used to endorse or promote products derived from this software without specific
+// prior written permission.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY
+// EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+// OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
+// SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+// INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
+// TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR
+// BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+// CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+// ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH
+// DAMAGE.
+//
+// HISTORY:
+//           BST : 04/01/2021 Initial Release
+//
+
+
+#ifndef _Basler_mE5AQ8CXP6D_H_
+#define _Basler_mE5AQ8CXP6D_H_
+#pragma once
+#include "../../../MMDevice/MMDevice.h"
```

Comment:
Please get rid of `../../../MMDevice/`. This folder is already set as an include directory in the project and we don't won't to assume a fixed relative path to this directory.

This should look just like `#include "MMDevice.h"`